<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>itemWidget</class>
 <widget class="QWidget" name="itemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>277</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="leIcon">
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>44</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>55</width>
       <height>46</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;宋体&quot;;</string>
       </property>
       <property name="text">
        <string>本地传输-20161220105212</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>8</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 8pt &quot;宋体&quot;;
color: rgb(170, 170, 170);</string>
       </property>
       <property name="text">
        <string>完成0个，最近：12/20 10:45 &lt;0&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="le_gif">
     <property name="minimumSize">
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>51</width>
       <height>51</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
